docusaurus/docs-internals/editor-controls/multivalue-controls/text-shadow.md

```diff
@@ -0,0 +1,164 @@
+| `type`      | `string`                                 |      -       | Type should be `"textShadow"` to use this control                                                                                                                                                                                                                                                                                       |
+| `position?` | `number`                                 |      -       | The position of the control in toolbar                                                                                                                                                                                                                                                                                                  |
+| `devices?`  | `"all"` \| `"desktop"` \| `"responsive"` |   `"all"`    | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices                                                                                                        |
+| `disabled?` | `boolean`                                |   `false`    | Configure the condition under which the control is disabled or enabled                                                                                                                                                                                                                                                                  |
+| `states?`   | `Array<State>`                           | [`"normal"`] | Allows for different styles based on the element's state <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination) |
+| `selector?` | `string`                                 |      -       | Selector that determines the target element to which the text shadow styling will be applied                                                                                                                                                                                                                                            |
+
+### Return value
+
+Returns an object with the following values:
+
+`blur` - specifies the blur radius for the shadow; <br/>
+`hex` - defines the color of the shadow in hexadecimal format; <br/>
+`horizontal` - horizontal offset of the shadow; <br/>
+`opacity` - indicates the opacity of the text itself, not the shadow; <br/>
```

Comment:
opacity - indicates the opacity of the text itself, not the shadow; -> opacity - indicates the opacity of the shadow;